sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -88,4 +88,10 @@
   Integer getCassandraPort();
   void setCassandraPort(Integer port);
 
+  /* Text */
+  @Description("Lines of text count")
+  @Default.Long(100000)
+  Long getLinesOfTextCount();
+
+  void setLinesOfTextCount(Long count);
```

Comment:
<!--new_thread; commit:c6c7070ad92424707d3720d3a4dc2c0fb6961440; resolved:0-->
Will it make sense to change this property to a generic name for all file based sources/sinks  (for example numberOfRecords). If we do this we loose the ability to specify different defaults etc for different file formats but I feel like it might be OK to do this in favor of simplifying the options class.

Reply:
Ok, I agree.